src/material/core/tokens/m2/mat/_select.scss

```diff
@@ -17,6 +17,7 @@ $prefix: (mat, select);
   $on-surface: if($is-dark, #fff, #000);
 
   @return (
+    list-item-selected-container-color: rgba($on-surface, if($is-dark, 0.08, 0.04)),
```

Comment:
The option already has a `selected-state-layer-color` token for this. Also I _think_ the option selected state is correct atm, we were discussing this yesterday with @mmalerba

Reply:
Are we certain the option selected state color is correct? This is what the background-color looks like without this override:

![Screenshot 2024-02-01 at 2 24 36 PM](https://github.com/angular/components/assets/25158423/f84f6885-0ba5-448d-9a70-0b84079bacaf)

And this is what it looks like with it:

![Screenshot 2024-02-01 at 2 24 15 PM](https://github.com/angular/components/assets/25158423/8fdf48d3-0f49-4dcb-ba97-1bb7f494f664)

Reply:
Yeah so my understanding is that this is more a quirk of our dev app than an issue with the current setup. The second screenshot is what it'll look like for users by default. @mmalerba has some updates for the dev app in #28516.